src/components/Grid/GridComponent.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div id="grid">
+    <table
+      v-if="treeSelected!=-1"
+    >
+      <tr>
+        <th>Variable</th>
+        <th :colspan="gridAssessments.length + 1">Collection Round</th>
+      </tr>
+      <tr>
+        <th class="w-0"></th>
+        <th class="w-0"></th>
+        <th
+          v-for="assessment in gridAssessments"
+          :key="assessment.id"
+          class="text-center">
+          {{assessment.name}}
+        </th>
+      </tr>
+      <tr>
+        <th></th>
+        <td>
+          <facet-option class="selectAll">All</facet-option>
+        </td>
+        <td v-for="assessment in gridAssessments"
+            :key="assessment.id">
+          <facet-option class="selectCol"><font-awesome-icon icon="arrow-down" /></facet-option>
+        </td>
+      </tr>
+
+      <tr
+        v-for="(row, rowIndex) in grid"
+        :key="rowIndex"
+      >
+        <th>
+          <span class="title">
+            {{variables[rowIndex].label?variables[rowIndex].label:variables[rowIndex].name}}
+          </span>
+        </th>
+        <td>
+          <facet-option class="selectRow"><font-awesome-icon icon="arrow-right" /></facet-option>
+        </td>
+        <td :key="colIndex"
+            v-for="(cell,colIndex) in row">
+          <facet-option
```

Comment:
not sure if facetOption is best option for this , but i guess it works for now, suggest to move to dedicated cell

Reply:
agreed